opae.spec.rhel

```diff
@@ -138,7 +137,7 @@ cp binaries/opae.io/scripts/*.py %{buildroot}%{_usr}/src/opae/samples/opae.io/sc
 
 
 %if 0%{?rhel}
-  %make_install
+  %cmake_install
```

Comment:
This was per Tom.  Why are we changing it?

Reply:
changes from  %make_build  to %cmake_build because of genearte rpm fails on RHEL 9.0 version